sources/packages/web/src/components/fa-application/StringToHtmlParagraphs.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
This component should be in global scope

Reply:
I will move it. I was not sure it would be good to promote it as a global right now, but it is a quick change and I will do it.

Reply:
Done!